pkg/app/topaz.go

```diff
@@ -267,7 +267,7 @@ func mapToGRPCPorts(api map[string]*builder.API) map[string]services {
 	return portMap
 }
 
-func contains[T comparable](slice []T, item T) bool {
+func Contains[T comparable](slice []T, item T) bool {
```

Comment:
We should replace this function with [`lo.Contains`](https://github.com/samber/lo?tab=readme-ov-file#contains)